### PR TITLE
Update Complying with Licenses page

### PR DESCRIPTION
**Pull request by nathanfranke**
- Clarify that it is not legal advice.
- Remove outdated third-party attributions, and point them to COPYRIGHT.txt instead.

### CODE REVIEW

**Comment by nathanfranke** [comment]
Q: I am working on an [addon](https://github.com/nathanfranke/nicense) to simplify the attribution process. Would it be okay to add a note of it to this documentation?

**Comment by nathanfranke** [comment]
Updated and added some sample code on how to generate a licenses text label

File: about/complying_with_licenses.rst [hunk 1]
```diff
@@ -11,20 +17,25 @@ It doesn't have a sole owner either, as every contributor that submits code to
...
-text somewhere in your game or derivative project.
+For Godot itself, the only requirement is to include the license text somewhere in your game or derivative project.
 
 This text reads as follows:
 
-    This game uses Godot Engine, available under the following license:
+    This project uses Godot Engine, available under the following license:
 
     Copyright (c) 2007-2021 Juan Linietsky, Ariel Manzur.
     Copyright (c) 2014-2021 Godot Engine contributors.
```
Comment (mhilbrunner) [inline about/complying_with_licenses.rst:41]:
```suggestion
    Copyright (c) 2007-2022 Juan Linietsky, Ariel Manzur.
    Copyright (c) 2014-2022 Godot Engine contributors.
```

**Comment by nathanfranke** [comment]
I can't guarantee formatting checks will pass, so I'll need the workflows to be approved :slightly_smiling_face:

**Review by mhilbrunner — APPROVED** [review]
Changes look good to me. WDYT, @akien-mga?

**Comment by akien-mga** [comment]
There are some good updates in this page, but it also goes in the general direction of making the requirements very extreme, to a point where personally I'd skip using Godot and go for an engine with EULA just so that I don't have to deal with such requirements.

If the page starts by saying that I can't ship a game without consulting with an attorney, as a small indie dev, I would look for another engine that just asks me to keep a "Made with Unity" splash screen. (And I'm not saying that license compliance is *easier* with proprietary software - it's just that they don't care and nobody respects the license terms of e.g. FreeType which is included in Unity too. It's mentioned in their EULA, but not in any Unity game.)

Likewise, adding code examples on how to dump a ton of license text in people's credits is IMO not the right approach. It looks terrible, and that's not what most licenses require. _Personally_, I'm perfectly fine with folks linking to https://godotengine.org/license and letting _that page_ do the attribution properly.

Eventually, I also intend to add a way to embed the editor's About dialog in the game, in a way that can be shown easily with e.g. a hotkey (for example Alt+L, or whatever wouldn't conflict with typical OS and in-game hotkeys). Then complying with licenses would be as simple as:

```
Made with Godot Engine
https://godotengine.org
(Press Alt+L for license details.)
```

**Comment by Calinou** [comment]
> Eventually, I also intend to add a way to embed the editor's About dialog in the game, in a way that can be shown easily with e.g. a hotkey (for example Alt+L, or whatever wouldn't conflict with typical OS and in-game hotkeys). Then complying with licenses would be as simple as:

I started working on this a while ago, but didn't have time to complete it: https://github.com/Calinou/godot/tree/add-license-notices-gui

**Comment by nathanfranke** [comment]
There is also github.com/nathanfranke/nicense, written in gdscript so not suitable for the engine, but nevertheless Unlicense so totally free to steal

**Comment by mhilbrunner** [comment]
I'm going to close this for now due to the discussion outlined above. It may still be worthwhile to improve this page, but we should discuss what/how first. Thanks for taking the time with this PR :)

**Pull request closed without merging**